app/views/devise/confirmations/new.html.haml

```diff
@@ -0,0 +1,10 @@
+%h2 Resend confirmation instructions
```

Comment:
devise view should be translatable as well.
read https://github.com/heartcombo/devise/wiki/I18n